lib/ruby_lsp/requests/code_lens.rb

```diff
@@ -146,6 +151,30 @@ def add_code_lens(node, name:, command:)
           type: "debug",
         )
       end
+
+      sig { params(node: SyntaxTree::Command).returns(T.nilable(String)) }
+      def resolve_gem_homepage(node)
+        gem_name = node.arguments.parts.flat_map(&:child_nodes).first.value
+        spec = Gem::Specification.stubs.find { |gem| gem.name == gem_name }&.to_spec
+        return if spec.nil?
+
+        spec.homepage || spec.metadata.fetch("homepage_uri", nil)
+      end
+
+      sig { params(node: SyntaxTree::Command, homepage: String).void }
+      def add_open_gem_homepage_code_lens(node, homepage:)
+        range = range_from_syntax_tree_node(node)
+
+        @response << Interface::CodeLens.new(
+          range: range,
+          command: Interface::Command.new(
+            title: "Open Homepage",
+            command: "rubyLsp.openGemHomepage",
+            arguments: [homepage],
+          ),
+          data: { type: "browser" },
```

Comment:
We may want to link to other things in the editor eventually.
```suggestion
          data: { type: "link" },
```